vignettes/imputation.Rmd

```diff
@@ -28,31 +45,19 @@ Relative Day) are numeric variables. They can be derived only if the date or
 datetime is complete. Therefore `{admiral}` provides imputation functions which fill
 in missing date or time parts according to certain imputation rules.
 
-In {admiral} we use only two functions `derive_vars_dt()` and
-`derive_vars_dtm()` for date and datetime imputations respectively. In all other
-functions where dates can be passed as an argument, we expect full dates or
-datetimes (unless otherwise specified), so if any possibility of partials then
-these functions should be used as a first step to make the required imputation.
+In `{admiral}` users will primarily use two functions `derive_vars_dt()` and `derive_vars_dtm()` for date and datetime imputations respectively. In all other functions where dates can be passed as an argument, we expect full dates or datetimes (unless otherwise specified), so if any possibility of partials then these functions should be used as a first step to make the required imputation.
 
-## Required Packages
+The functions that need to do date/time imputation follow a rule that we have called **Highest Imputation**, which has a corresponding argument in all our functions called `highest_imputation`. This rule allows a user to control which components of the DTC value are imputed if they are missing. 
```

Comment:
I think we should not only mention that this rule exists but also explain what it means.

Reply:
I put in a pretty weak argument.  

I started writing and then making a table and was left with more questions then answers.  I think whatever we make in this vignette to help explain the rule we should then populate into the function documentation as I was getting so confused on how to explain it.  Unless you have a pithy phrase?

Going to make a new issue for this update

Reply:
https://github.com/pharmaverse/admiral/discussions/2090